.gitignore

```diff
@@ -1,3 +1,4 @@
 target/
 **/*.rs.bk
 wisegui/Cargo.lock
+*.json
```

Comment:
Why was this added? Is this needed by anything?

Reply:
I use VS code and it adds json files for some settings.